packages/core/src/App/Constants/header-links.js

```diff
@@ -8,14 +8,14 @@ export const header_links = [
     {
         id: 'dt_reports_tab',
         icon: <Icon icon='IcReports' className='header__icon' />,
-        text: localize('Reports'),
+        text: () => localize('Reports'),
```

Comment:
nit: could using `<Localize />` pass in the translation correctly?

Reply:
@kelly-binary it would have to be `() => <Localize i18n_default_text="..."` in that case because the Localize component doesn't rerender if the text prop passed to it doesn't change. 
(on the first render the language.json files hasn't been loaded and the text will remain as english).

Unless there is a different way to handle this? 🤔

Reply:
Ahh, I see 👍